recipes/shadows/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "shadows" %}
+{% set version = "0.1a2" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
@ocefpaf: Thank you for the pointer. I hope my latest commit addresses this properly. If not, please le me know what I did wrong. 😅